pkg/providers/vsphere/vsphere_test.go

```diff
@@ -2833,6 +2833,30 @@ func TestSetupAndValidateCreateClusterDefaultTemplate(t *testing.T) {
 	}
 }
 
+func TestSetupAndValidateCreateClusterSetDefaultTemplate(t *testing.T) {
```

Comment:
Don't you need to change the name of this function? We can say CP and the above Worker or something